### DISCUSSION TITLE
I made an iOS shortcut for Hoarder

### DISCUSSION THREAD

**Discussion by z-hhh** [karakeep-app/karakeep#91] [Show and tell]
Only link type for now.

https://www.icloud.com/shortcuts/af8e66cfcd034e3386f5e668374f9b63

**Comment by MohamedBassem** [comment]
That's very cool, thanks for sharing! I initially started with a shortcut as well (https://www.icloud.com/shortcuts/78734b46624c4a3297187c85eb50d800). It was much simpler than yours, but then ended up removing it in https://github.com/MohamedBassem/hoarder-app/commit/d3be75deb8784afdb123c4a1f3a7db4bd6f3c87d once the mobile app was developed. I'm curious, is there any reason you prefer using the shortcut over the mobile app?

**Reply by z-hhh** [reply]
I don't know if there is already one 🤣, otherwise I won't write it. I prefer using the shortcut just because it won't make an extra jump

**Reply by MohamedBassem** [reply]
Actually with the app, you can add it to the share sheet as well and it would be as fast

![image](https://github.com/MohamedBassem/hoarder-app/assets/2418637/f8e18763-04b7-4561-b4e7-e751170bf74f)

**Reply by simpzan** [reply]
ios version 1.3.2, save a link from safari will jump to the Hoarder app. the experience is not good. a better interaction should be staying on the safari, like Pocket.

**Reply by huyz** [reply]
Would love to see something like then Evernote iOS app's pop-up dialog that lets you select tags when sharing to it

**Comment by huyz** [comment]
Thanks, Very nice. 

Btw the config clearing doesn't work because the filename doesn't match
